code/modules/events/wizard/magicarp.dm

```diff
@@ -45,12 +45,6 @@
 	projectiletype = pick(allowed_projectile_types)
 	. = ..()
 
-/*/mob/living/simple_animal/hostile/carp/ranged/xenobio
-	desc = "45% magic, 50% carp, 5% slime, 100% horrible."
-	allowed_projectile_types = list( /obj/item/projectile/magic/animate, /obj/item/projectile/magic/teleport, /obj/item/projectile/magic/door, /obj/item/projectile/magic/aoe/fireball,
-	/obj/item/projectile/magic/spellblade, /obj/item/projectile/magic/arcane_barrage)
-	gold_core_spawnable = HOSTILE_SPAWN*/
-
```

Comment:
this tbh

Reply:
teleporting, fireballing, creating doors, just what the game needs

Reply:
not the ability to create doors